modules/tinymce/src/themes/silver/main/ts/ui/dropdown/CommonDropdown.ts

```diff
@@ -1,6 +1,22 @@
 import {
-  AddEventsBehaviour, AlloyComponent, AlloyEvents, AlloyTriggers, Behaviour, CustomEvent, Dropdown as AlloyDropdown, Focusing, GuiFactory, Keying,
-  Memento, Replacing, Representing, SimulatedEvent, SketchSpec, TieredData, Unselecting
+  AddEventsBehaviour,
+  AlloyComponent,
+  AlloyEvents,
+  AlloyTriggers,
+  Behaviour,
+  CustomEvent,
+  Dropdown as AlloyDropdown,
+  Focusing,
+  GuiFactory,
+  Highlighting,
+  Keying,
+  Memento,
+  Replacing,
+  Representing,
+  SimulatedEvent,
+  SketchSpec,
+  TieredData,
+  Unselecting
 } from '@ephox/alloy';
```

Comment:
- [ ] reformat line

Reply:
The formatting here still isn't quite right sorry. You've now got one massively long line here and below you've still got things chopping on every import.

Reply:
I think TypeScript hero probably did this before I uninstalled it, and I don't seem to have any tools that automate the formatting for me. Is there a rule I should be following? Wrap at 160 characters?

Reply:
Ahh that's a shame that VSCode doesn't offer anything for that 😢 I wonder if we can use a lint rule for this 🤔 
 Anyways yeah that's right, for example the old version was this (it looks like it might have been wrapped at 150 at one stage but 160 is what is currently configured in eslint for max line length):
```ts
import {
  AddEventsBehaviour, AlloyComponent, AlloyEvents, AlloyTriggers, Behaviour, CustomEvent, Dropdown as AlloyDropdown, Focusing, GuiFactory, Keying,
  Memento, Replacing, Representing, SimulatedEvent, SketchSpec, TieredData, Unselecting
} from '@ephox/alloy';
```

Reply:
Copied the origin/develop and manually inserted Highlighting. I'll need to get whatever vscode setup everyone else has, although I suspect this was more a problem because I started using TypeScript Hero

Reply:
Looks good thanks.